toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/internal/components/utilitynetwork/AddAssociationFromSourceViewModel.kt

```diff
@@ -305,13 +434,18 @@ internal class AddAssociationFromSourceViewModel(
      */
     fun selectSource(source: UtilityAssociationFeatureSource?) {
         _selectedSource.value = source
+        // Clear previously set filter text
+        setFeatureSourcesFilterText("")
```

Comment:
```suggestion
        setAssetTypesFilterText("")
```
I think this should be clearing the asset type filter text.

Reply:
@puneet-pdx This is still clearing both the featureSourcesFilterText and assetTypesFilterText

Reply:
updated